yosys/equiv_simple_clk2fflogic.yosys

```diff
@@ -0,0 +1,10 @@
+hierarchy
```

Comment:
Does Yosys support comments in the script files?

If so could we get one at the top of each of these files?

Reply:
Done